### PR TITLE
Enable momentum operator for a ngraph engine

### PR DESCRIPTION
**Pull request by mozga-intel**
Introduces the nGraph's implementation of the momentum operator.
test=develop

### CODE REVIEW

**Comment by mozga-intel** [comment]
@luotao1, @tensor-tang Can you please continue review and comment this pull-request?. These operators are crucial for us.

File: python/paddle/fluid/tests/unittests/ngraph/test_momentum_ngraph_op.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle.fluid.core as core
+from paddle.fluid.op import Operator
+from paddle.fluid.tests.unittests.op_test import OpTest
```
Comment (tensor-tang) [inline python/paddle/fluid/tests/unittests/ngraph/test_momentum_ngraph_op.py]: Can you reuse https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/test_momentum_op.py , it seems much code is duplicated.

Comment (mozga-intel) [reply python/paddle/fluid/tests/unittests/ngraph/test_momentum_ngraph_op.py]: @tensor-tang Done.

**Review by tensor-tang — APPROVED** [review]
LGTM

**Pull request merged**